content/foundation/voting.md

```diff
@@ -117,10 +116,13 @@ for release, and there must be more positive than negative votes.
 Generally the community
 will cancel the release vote if anyone identifies serious problems, but
 in most cases the ultimate decision
-lies with the individual serving as release manager. The
+lies with the individual serving as release manager (RM). The
```

Comment:
In terms of writing style on this website, I don't think we use acronyms generally, if we can avoid them. I'd rather have "release manager" everywhere.

Reply:
Make sense.